### ISSUE TITLE
Fields are missing in rabbitmq with LS 2.0 (plugin version 3.0.5)

### ISSUE THREAD

**Issue by jakommo** [logstash-plugins/logstash-output-rabbitmq#35]
With LS 2.0 and plugin version 3.0.5, only the message is pushed to rabbitmq but not the other fields.
In rubydebug the fields show up but are missing in rabbitmq.

Example config:

```
input {
  file {
    sincedb_write_interval => 5
    path => [ "/home/jakommo/ELASTIC/ISE/14912/test/messages" ]
    type => "messages"
    sincedb_path => "/dev/null"
    start_position => "beginning"
  }
}

filter {
  if [type] == "messages" {
     grok {
       match  => [ "message", "%{NUMBER:priority} %{SYSLOGTIMESTAMP:syslog_timestamp} %{SYSLOGHOST:syslog_hostname} %{DATA:syslog_program}(?:\[   %{POSINT:syslog_pid}\])?: %{GREEDYDATA:syslog_message}" ]
       }
  }
}

output {
  stdout { codec => rubydebug }
  rabbitmq {
        exchange        => "logstash_exchange"
        host            => "localhost"
        exchange_type   => "direct"
        key             => "logstash"
        durable         => true
        persistent      => true
        user            => "XXX"
        password        => "XXX"
  }
}
```

Example log line:

```
6 Nov 1 15:47:29 host01 ntpd[47533]: 0.0.0.0 c059 01 restart
```

Payload in rabbitmq:
Loaded with 1.5.4 (plugin version 1.1.1)

```
{"message":"6 Nov 1 15:47:29 host01 ntpd[47533]: 0.0.0.0 c059 01 restart ","@version":"1","@timestamp":"2015-11-03T11:30:31.156Z","host":"es-jre","path":"/home/jakommo/ELASTIC/ISE/14912/test/messages","type":"messages","priority":"6","syslog_timestamp":"Nov 1 15:47:29","syslog_hostname":"host01","syslog_program":"ntpd[47533]","syslog_message":"0.0.0.0 c059 01 restart "}
```

Loaded with 2.0 (plugin version 3.0.5)

```
2015-11-03T11:31:13.379Z es-jre 6 Nov 1 15:47:29 host01 ntpd[47533]: 0.0.0.0 c059 01 restart 
```

In both cases rubydebug is:

```
{
             "message" => "6 Nov 1 15:47:29 host01 ntpd[47533]: 0.0.0.0 c059 01 restart ",
            "@version" => "1",
          "@timestamp" => "2015-11-03T11:31:13.379Z",
                "host" => "es-jre",
                "path" => "/home/jakommo/ELASTIC/ISE/14912/test/messages",
                "type" => "messages",
            "priority" => "6",
    "syslog_timestamp" => "Nov 1 15:47:29",
     "syslog_hostname" => "host01",
      "syslog_program" => "ntpd[47533]",
      "syslog_message" => "0.0.0.0 c059 01 restart "
}
```

So the fields are parsed correctly but not pushed to rabbitmq

**Comment by andrewvc** [comment]
@jakommo the issue here seems to be that before the default codec was JSON, and that no longer is the case. I do think we should fix this and restore the `json` codec.

A workaround for now would be to explicitly set the codec to `json`.

**Comment by andrewvc** [comment]
Fixed by #36
